project_task_stage_change_restriction/readme/CONTEXT.md

```
@@ -0,0 +1 @@
+There may be some cases where you would like to prevent certain users from moving a task into certain stages. For example developers can move tasks to the "Review" stage, but only project manager can move a task to the "Done" or "Cancel" stages.
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)